src/Components/SignUp/signUp.css

```diff
@@ -0,0 +1,114 @@
+.signUp-form {
+  display: grid;
+  grid-template-areas:
+    "fname lname"
+    "finput linput"
+    "emaillabel passlabel"
+    "email pass"
+    "confpasslabel -"
+    "confpass -"
+    "btn1 btn2";
+
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: 0fr 0fr 0fr 0fr 0fr 0fr;
```

Comment:
you could just use `grid-template-rows: repeat(no-of-rows, row-width)` if the number of rows is fixed.